src/analysis/retail/shaman/restoration/modules/talents/FlashFlood.tsx

```diff
@@ -63,6 +63,8 @@ class FlashFlood extends Analyzer {
   constructor(options: Options) {
     super(options);
     this.active = this.selectedCombatant.hasTalent(TALENTS.FLASH_FLOOD_TALENT.id);
+    this.flashFloodHaste =
+      this.selectedCombatant.getTalentRank(TALENTS.FLASH_FLOOD_TALENT.id) * 0.1;
```

Comment:
I would recommend turning the 0.1 into a constant instead. This way there will be no questionsi n the future what this multiplecation is for.